python/ray/tune/tests/test_actor_reuse.py

```diff
@@ -61,31 +58,6 @@ def reset_config(self, new_config):
     return MyResettableClass
 
 
-def create_resettable_function(num_resets: defaultdict):
-    def trainable(config, checkpoint_dir=None):
-        if checkpoint_dir:
-            with open(os.path.join(checkpoint_dir, "chkpt"), "rb") as fp:
-                step = pickle.load(fp)
-        else:
-            step = 0
-
-        while step < 2:
-            step += 1
-            with tune.checkpoint_dir(step) as checkpoint_dir:
-                with open(os.path.join(checkpoint_dir, "chkpt"), "wb") as fp:
-                    pickle.dump(step, fp)
-            tune.report(**{"done": step >= 2, "iter": step, "id": config["id"]})
-
-    trainable = wrap_function(trainable)
-
-    class ResetCountTrainable(trainable):
-        def reset_config(self, new_config):
-            num_resets[self.trial_id] += 1
-            return super().reset_config(new_config)
-
-    return ResetCountTrainable
-
-
```

Comment:
This test never worked, as `num_resets` was local to each remote actor.

There is no great way to count the number of resets for function trainables so I've removed this for now